### DISCUSSION TITLE
Change Detection in Hyperspectral Imagery (Project 210)

### DISCUSSION THREAD

**Discussion by robertogl** [mathworks/MATLAB-Simulink-Challenge-Project-Hub#42] [Collaborate]
Contribute to the discussion by asking and/or answering questions, commenting, or sharing your ideas for solutions to [project 210](https://github.com/mathworks/MathWorks-Excellence-in-Innovation/tree/main/projects/Change%20Detection%20in%20Hyperspectral%20Imagery)

---
*:robot: An AI bot may reply to your comments with suggestions and guidance. It will not solve the project for you. Your comments may be processed by a third-party LLM. To direct a question to a specific person, tag them with @username.*

**Comment by JaidevSK** [comment]
Hi, I am Jaidev, a B.Tech student from Indian Institute of Technology (IIT) and I am interested in working and collaborating on this project.
